src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -1176,6 +1176,79 @@
+
+helps['storage share-rm exists'] = """
+type: command
+short-summary: Check for the existence of an Azure file share.
+examples:
+  - name: Check for the existence of an Azure file share 'MyFileShare' under the storage account 'MyStorageAccount' (account name) in resource group 'MyResourceGroup'.
+    text: az storage share-rm exists -g MyResourceGroup --storage-account MyStorageAccount --name MyFileShare
+  - name: Check for the existence of an Azure file share 'MyFileShare' under the storage account 'MyStorageAccount' (account id).
+    text: az storage share-rm exists --storage-account MyStorageAccount --name MyFileShare
+  - name: Check for the existence of an Azure file share by resource id.
+    text: az storage share-rm exists --ids file-share-id
+"""
+
+helps['storage share-rm list'] = """
+type: command
+short-summary: List the Azure file shares under the specified storage account.
+examples:
+  - name: List the Azure file shares under the storage account 'MyStorageAccount' (account name) in resource group 'MyResourceGroup'.
+    text: az storage share-rm list -g MyResourceGroup --storage-account MyStorageAccount
+  - name: List the Azure file shares under the storage account 'MyStorageAccount' (account id).
+    text: az storage share-rm list --storage-account MyStorageAccount
+"""
+
+helps['storage share-rm show'] = """
+type: command
+short-summary: Show the properties for a specified Azure file share.
+examples:
+  - name: Show the properties for an Azure file share 'MyFileShare' under the storage account 'MyStorageAccount' (account name) in resource group 'MyResourceGroup'.
+    text: az storage share-rm show -g MyResourceGroup --storage-account MyStorageAccount --name MyFileShare
+  - name: Show the properties for an Azure file share 'MyFileShare' under the storage account 'MyStorageAccount' (account id).
+    text: az storage share-rm show --storage-account MyStorageAccount --name MyFileShare
+  - name: Show the properties of an Azure file shares by resource id.
+    text: az storage share-rm show --ids file-share-id
+"""
+
+helps['storage share-rm update'] = """
+type: command
+short-summary: Update the properties for an Azure file share.
+examples:
+  - name: Update the properties for an Azure file share 'MyFileShare' under the storage account 'MyStorageAccount' (account name) in resource group 'MyResourceGroup'.
+    text: az storage share-rm update -g MyResourceGroup --storage-account MyStorageAccount --name MyFileShare --share-quota 3 --metadata key1=value1 key2=value2
+  - name: Update the properties for an Azure file share 'MyFileShare' under the storage account 'MyStorageAccount' (account id).
+    text: az storage share-rm update --storage-account MyStorageAccount --name MyFileShare --share-quota 3 --metadata key1=value1 key2=value2
+  - name: Update the properties for an Azure file shares by resource id.
+    text: az storage share-rm update --ids file-share-id --share-quota 3 --metadata key1=value1 key2=value2
+"""
+
```

Comment:
storage account name must be lower case. can you change it for examples?

Reply:
OK, will update it

---
In reply to: [367800069](https://github.com/Azure/azure-cli/pull/11649#discussion_r367800069) [](ancestors = 367800069)